cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,123 @@
+
+- **Routing** - Once we have clarified any questions we have on a submitted ticket and we are able to replicate the problem internally, the ticket is ready to be routed to the appropriate team at Cypress.
+
+- **Prioritization** - Teams meet regularly to review tickets that have been routed to them via triage. They evaluate each ticket based on the a number of criteria outlined in our [prioritization rubric](#how-we-prioritize-issues). Each dimension of our prioritization rubric carries a weight. The following equation is then used to give us a **relative priority**. The relative priority score is then divided by the estimated effort to address the ticket to give us a **priority-to-effort** score which we then use to determine which tickets are highest priority. 
+**Important note - These priorities do not dictate when a team will be able to start work. This simply gives teams the data they need to make informed prioritization choices.**
+
+- **Resolution and Verification** - Once an issue has been picked up for work, it will have the label **Stage: Under Development** attached. Developers will then begin work on the issue. Sometimes the scope of the work will be greater than the ticket submitted, and a parent ticket will be created to encapsulate the entire scope of the work. A link to the original ticket will be created in the new ticket. 
+
+- **Release** - Once work has been finished, the ticket will be scheduled for release. If the changes are non-breaking, we generally aim to include the work in the next minor release every-other Tuesday. If the changes made to address the issue require breaking changes, the issue will be scheduled for our next major release roughly every quarter.
+
+
+## How We Prioritize Issues
+
+At Cypress, we use the following guidelines to help us standardize the importance of every ticket submitted. Of course there are subjective interpretations for each of these fields, but the goal is that each ticket is considered and examined thoroughly in a standardized way. Once values have been determined for each field, a total priority is calculated. It is important to note that these values are **a guide** to making informed decisions around what issues bring the most value to the community and Cypress as an organization. They are relative and fuzzy and not to be treated as gospel. It is also important to understand that these values may change as future factors and circumstances change.
+
+Here are the criteria we use to gauge relative issue priority:
```

Comment:
Suggestion to add a subheading for prioritization rubric, since it's referenced with this particular wording a few times. If we add the subheading, we'll need to update the links for "prioritization rubric" to point here

```suggestion
### Prioritization Rubric

Here are the criteria we use to gauge relative issue priority:
```